### ISSUE TITLE
Add Persian  for dictionary

### ISSUE THREAD

**Issue by pserwylo** [lexica/lexica#63]
After speaking with one or two people, it seems that there is no reason this shoudln't work. GNU ASpell contains an `fa` dictionary in debian, and I can successfully use that to generate a word list. However for some reason the tools I created to generate random boards in Latin based languages is not working with this wordlist. I'll have to investigate further to figure out why this is the case.

**Comment by pserwylo** [comment]
I now have this working. Requires a tiny bit more testing on my end to make sure that it works as expected, then I'll release a new version which includes that dictionary.

I do have a question for any native speakers who may be interested. When adding a new language, Lexica needs to be populated with a "probability generator" thing. This is responsible for deciding which letters to randomly choose, in such a way that the resulting boards still have a reasonable amount of words on them.

The metric I'm using for deciding whether the resulting "probability generator" is appropriate is to randomly generate 1000 different 4x4 boards, and count how many words end up on each of these boards. I then look at the resulting statistics and ensure that:
 * There are not any boards which result in very few words (it is not fun to play a board with 0 words, and I personally don't think only 5 words would be very fun either).
 * There are not any boards which contain 1000s of words (otherwise you'd never even come close to getting all of the words in the required time).
 * The average number of words is suitable (in general, they seem to be just over 100 for most languages).

Here are the statistics for all of the other languages, when generating 1000 different 4x34 boards and then asking how many words on each board:

```
English (US)  Min:  8  Mean: 163  Max: 344  SD: 59
English (GB)  Min: 25  Mean: 133  Max: 308  SD: 46
German (DE)   Min:  8  Mean:  47  Max: 159  SD: 29
French (FR)   Min:  9  Mean:  69  Max: 249  SD: 41
Spanish       Min: 24  Mean: 176  Max: 445  SD: 97
Italian       Min: 17  Mean: 125  Max: 391  SD: 67
Dutch         Min: 15  Mean:  90  Max: 219  SD: 43
Catalan       Min: 19  Mean: 124  Max: 318  SD: 67
```

Generally, the algorithm I use to generate these probability distributions struggles because it chooses too few words. For example, you can see the German boards tend to have much fewer words. 

Persian has the opposite problem, consistently generating quite a high number of words, and never really generating boards with only a small number of words. I quite enjoy playing English boards with, e.g., 35 words on them, but here are the statistics for the person probability generators that my algorithm is generating (SD is standard deviation):

```
Min:  14 Mean: 190 Max: 550 SD: 85
Min:  34 Mean: 187 Max: 438 SD: 83
Min:  34 Mean: 203 Max: 468 SD: 80
Min:  35 Mean: 191 Max: 390 SD: 79
Min:  36 Mean: 185 Max: 461 SD: 84
Min:  36 Mean: 193 Max: 427 SD: 81
Min:  36 Mean: 216 Max: 516 SD: 96
Min:  38 Mean: 164 Max: 431 SD: 77
Min:  38 Mean: 189 Max: 422 SD: 80
Min:  38 Mean: 190 Max: 488 SD: 89
Min:  46 Mean: 203 Max: 395 SD: 78
Min:  49 Mean: 197 Max: 461 SD: 85
Min:  49 Mean: 211 Max: 486 SD: 84
Min:  52 Mean: 191 Max: 431 SD: 82
Min:  54 Mean: 218 Max: 449 SD: 92
Min:  55 Mean: 235 Max: 534 SD: 99
Min:  76 Mean: 266 Max: 624 SD: 98
Min:  77 Mean: 213 Max: 441 SD: 81
Min:  82 Mean: 241 Max: 601 SD: 100
Min: 102 Mean: 243 Max: 523 SD: 96
```

What this means, is that if I chose the bottom probability generator (Min: 102, Mean: 243, Max: 523) and put it into Lexica, then essentially no Persian board would have less than 100 words on it.

Is there something about the persian lagnuage and script which would make this desirable (e.g. you could anticipate generating many words really quickly)? Or should I intentionally choose one of the probability generators which produces fewer words (e.g. Min:  14 Mean: 190 Max: 550 SD: 85)?

I hope that makes sense...

**Comment by pserwylo** [comment]
This is now available in version 0.11.2 on F-Droid.

I still need to fix the "Letter Points" scoring option which will crash, however that needs to be fixed for all languages.
